plugin/src/helpers/config.ts

```diff
@@ -24,13 +37,19 @@ export const getNextConfig = async function getNextConfig({
   failBuild = defaultFailBuild,
 }): Promise<NextConfig> {
   try {
-    const { config, appDir, ignore }: RequiredServerFiles = await readJSON(join(publish, 'required-server-files.json'))
+    const { config, appDir, ignore, files }: RequiredServerFiles = await readJSON(
+      join(publish, 'required-server-files.json'),
+    )
     if (!config) {
       // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
       return failBuild('Error loading your Next config')
     }
-    return { ...config, appDir, ignore }
+
+    const routesManifest = (await readJSON(files.find((f) => f.endsWith(ROUTES_MANIFEST_FILE)))) as RoutesManifest
+
+    // If you need access to other manifest files, you can add them here as well
+    return { ...config, appDir, ignore, routesManifest }
```

Comment:
I expose this through `getNextConfig(...)` so it's easily accessible, but so far it's only used in the `onPostBuild` step of the plugin. Originally, I had that same code in the `onPostBuild` instead of encapsulating it in `getNextConfig(...)`.